browser/js/global.js

```diff
@@ -904,11 +905,6 @@ function getInitializerClass() {
 
 	};
 
-	if (!global.prefs.getBoolean('hasNavigatorClipboardWrite', true)) {
```

Comment:
This more or less reverts a161ee30f471c5cd00e17428bd850c2a8604fe32, which was a fix for https://github.com/CollaboraOnline/online/issues/9072, this would mean that e.g. in ownCloud richdocuments (or any other non-updated integration) copy would fail for the first time you edit a document, and only the second copy would work.

The upside would be that in case the integration was broken once, but fixed today, it would start to work.

Do you think that's an acceptable trade-off? It feels a bit brave.

May better to keep the setting, but rename the key name, so we give a one-off chance for updated integrations?